src/components/navbar/navbar.module.scss

```diff
@@ -0,0 +1,77 @@
+.navbar {
+  height: 80px;
+  margin-left: 16px;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+
+.menu-bars {
+  margin-left: 2rem;
+  font-size: 2rem;
+  background: none;
+}
+
+.nav-menu {
+  background-color: #3d3f46;
+  width: inherit;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  transition: 850ms;
```

Comment:
This cannot be fixed. This makes it some sort of floating element

The 2nd AC: 
- The sidebar will take up space in the DOM
    - i.e. The sidebar should not be floating. The main content should move to make space for the side bar.